.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -0,0 +1,24 @@
+---
+name: ✨ Feature request
+about: Request a feature
+title: ''
+labels: ["C-request"]
+assignees: ''
+
+---
+
+<!--
+Note: Please search to see if an issue already exists for this request, or if the feature already exists.
+-->
+
+## Feature
+What is the feature you're requesting?
+
+## Why
+Why should your feature be added?
+
+## How
+If possible, describe how the proposal could be added.
+
```

Comment:
IMO this field should be removed, A person asking for a feature and then describing how to add that feature is more a proposal in my eyes